robot/src/main/java/org/frcteam2910/common/robot/drivers/Limelight.java

```diff
@@ -85,6 +138,11 @@ public void setCamMode(CamMode mode) {
         }
     }
 
+    /*
+     * Set the mode of the LED's of the Limelight. The options are DEFAULT, which sets it to whatever is specified in the pipeline,
```

Comment:
Instead of specifying what the options do in the method docs, add docs to the enum

Reply:
done